src/components/EditMenu.vue

```diff
@@ -347,6 +563,43 @@ const availableMiniWidgetTypes = computed(() =>
   }))
 )
 
+const pickVehicleImage = computed(() => {
+  switch (store.currentProfile.name.toLowerCase()) {
+    case 'rov default':
+      return '../src/assets/vehicles/BlueROV_thumb.png'
```

Comment:
Should we use `import` keys for this images to avoid possible breaks ? Since images path changes will be detected in compile time. At the moment this is not possible because of `computed`.
```ts
import RovThumb from '@/src/assets/vehicles/BlueROV_thumb.png'
```